### ISSUE TITLE
Seednode logs unhandled exception 10 times per second running master

### ISSUE THREAD

**Issue by wiz** [bisq-network/bisq#4110]
Logs are filled with this message 10 times per second:

```
Apr-01 16:20:54.588 [SeedNodeMain] ERROR b.c.s.CommonSetup: Uncaught Exception from thread SeedNodeMain
Apr-01 16:20:54.588 [SeedNodeMain] ERROR b.c.s.CommonSetup: throwableMessage= com.google.common.util.concurrent.Futures.addCallback(Lcom/google/common/util/concurrent/ListenableFuture;Lcom/google/common/util/concurrent/FutureCallback;)V
Apr-01 16:20:54.588 [SeedNodeMain] ERROR b.c.s.CommonSetup: throwableClass= class java.lang.NoSuchMethodError
Apr-01 16:20:54.589 [SeedNodeMain] ERROR b.c.s.CommonSetup: Stack trace:
java.lang.NoSuchMethodError: com.google.common.util.concurrent.Futures.addCallback(Lcom/google/common/util/concurrent/ListenableFuture;Lcom/google/common/util/concurrent/FutureCallback;)V
        at bisq.network.p2p.network.NetworkNode.sendMessage(NetworkNode.java:288)
        at bisq.network.p2p.peers.BroadcastHandler.sendToPeer(BroadcastHandler.java:172)
        at bisq.network.p2p.peers.BroadcastHandler.lambda$broadcast$2(BroadcastHandler.java:157)
        at bisq.common.FrameRateTimer.run(FrameRateTimer.java:50)
        at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
        at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:658)
        at bisq.common.MasterTimer$1.lambda$run$0(MasterTimer.java:38)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1135)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:844)

```

**Comment by freimair** [comment]
I tried to reproduce your issue with the latest master and have not been able to do so.

did you do a `./gradlew clean :seednode:installDist`? because your installer does a `build` only, which might not be enough...

if yes, may I ask for the logs from the startup to the error case you already postet?

**Comment by wiz** [comment]
hmm not sure why that fixed it, was a fresh clean VM and the master build was the only build it ever did so nothing to "clean" but it's not reproducing anymore

**Comment by wiz** [comment]
Fixed by #4116
